internal/common/validations.go

```diff
@@ -3,7 +3,38 @@
 
 package common
 
+import (
+	"strings"
+
+	"github.com/go-openapi/swag"
+)
+
 // IsAgentCompatible checks if the given agent image is compatible with what the service expects.
 func IsAgentCompatible(expectedImage, agentImage string) bool {
 	return agentImage == expectedImage
 }
+
+var NonIgnorableHostValidations []string = []string{"connected", "has-inventory", "machine-cidr-defined", "hostname-unique", "hostname-valid"}
+var NonIgnorableClusterValidations []string = []string{"api-vips-defined", "ingress-vips-defined", "all-hosts-are-ready-to-install", "sufficient-masters-count", "pull-secret-set", "cluster-preparation-succeeded"}
+
+var MayIgnoreValidation = func(validationID string, nonIgnorables []string) bool {
```

Comment:
And this one has no callers outside the package, it should be privat

Reply:
The private thing..it will have callers outside of the package when the next PR is applied, so if I make it private now, I have to change it later.

Reply:
> [This](https://github.com/openshift/assisted-service/pull/4927#discussion_r1094817565) comment is in a resolved conversation but it was not resolved
I have unresolved so that we can discuss, we can make this private, only to make it public in the next PR, seems fruitless to me.

Reply:
ok